systers_portal/meetup/models.py

```diff
@@ -42,9 +42,13 @@ def __str__(self):
         return self.title
 
     def clean_fields(self, *args, **kwargs):
-        """Validate meetup date is not less than today's date"""
+        """Validate meetup date is not less than today's date and meetup time is not a time
+        that has already passed."""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+        if self.time:
```

Comment:
Great catch on this bug!
Looking at this code, I realized we can make it better. Instead of keeping the validation logic in model `clean_fields()` it is actually better if we move it to the form and make those checks inside `clean_date()` and `clean_time()`. The difference that this change will make is that the form errors will be rendered as field errors (right beneath the field input). Currently those are rendered as generic `__all__` form errors.

Take your time to familiarize yourself with model vs form validation. Shout if anything not clear.